src/app/view/components/side-navigation-menu/side-navigation-menu.component.ts

```diff
@@ -74,7 +79,38 @@ export class SideNavigationMenuComponent implements AfterViewInit, OnDestroy {
     }
   }
 
-  constructor(private elementRef: ElementRef) {}
+  constructor(
+    private elementRef: ElementRef,
+    private getAllMenuSystemMenu: GetAllMenuSystemMenuUsecase
+  ) {}
+  ngOnInit(): void {
+    this._items = navigation.map((item) => {
```

Comment:
Qual o motivo para _items ainda existir?

Reply:
Esse objeto carrega a lista estática que está contida dentro do app-navigation.ts
Deixei ele pq não definimos como isso funcionaria definitivamente. Então ele contem a informação padrão da listagem do menu, porém se algo for carregado a partir da API, essa informação terá prioridade em relação ao valor padrão.